dockstore-webservice/src/main/java/io/dockstore/webservice/resources/AbstractWorkflowResource.java

```diff
@@ -447,8 +462,19 @@ private LambdaEvent createBasicEvent(String repository, String gitReference, Str
                 return updatedServices;
             }
             final DescriptorLanguageSubclass subclass = service.getSubclass();
+            final Boolean publish = service.getPublish();
+
             Workflow workflow = createOrGetWorkflow(Service.class, repository, user, "", subclass.getShortName(), gitHubSourceCodeRepo);
             workflow = addDockstoreYmlVersionToWorkflow(repository, gitReference, dockstoreYml, gitHubSourceCodeRepo, workflow);
+
+            if (publish != null) {
+                try {
+                    workflow = publishWorkflow(workflow, publish);
+                } catch (CustomWebApplicationException ex) {
+                    LOG.warn(ex.getMessage());
```

Comment:
(Copied from other comment)

Log the exception proper as well.

Hmm, the user will have no way of knowing this failed. Can/should we add it to GitHub app logs? I realize it will probably be just as easy to just see if something is published or not, but if it's in the logs, it gives us the option of doing something in the future.